disk/tests/common.py

```diff
@@ -15,3 +15,46 @@
     DEFAULT_DEVICE_NAME = '/dev/sda1'
     DEFAULT_FILE_SYSTEM = 'ext4'
     DEFAULT_MOUNT_POINT = '/'
+
+EXPECTED_METRICS = [
+    {"metric": "system.disk.free", "device": "tmpfs"},
+    {"metric": "system.fs.inodes.total", "device": "shm"},
+    {"metric": "system.fs.inodes.free", "device": "osxfs"},
```

Comment:
This metric is OSX only ?
Travis is not happy with it

Reply:
Yes, it looks like that